pkg/status/combinedstatus-resolution.go

```diff
@@ -586,22 +617,22 @@ func handleCombinedFields(scName string, scData *statusCollectorData) *v1alpha1.
 	if len(scData.GroupBy) > 0 {
```

Comment:
FYI, I find the method name "handleCombinedFields" to be not very clear about what this method is supposed to do. If you do not want to revise the name then I would find a comment on the method to be helpful.